addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -31,39 +29,122 @@
                 <default>60</default>
                 <advanced>true</advanced>
             </parameter>
+            <parameter name="TRACE_LOG" type="boolean" required="false">
```

Comment:
No, please do not introduce such parameters.
openHAB has one logging framework and mechanisms to configure it - bindings should not start to come up with their own stuff.